_includes/footer.html

```diff
@@ -47,7 +47,7 @@ <h6>Social Media</h6>
 
       <div class="row">
         <div class="col-md-6">
-          © 2011–2018 openSUSE contributors. License: <a href=https://www.gnu.org/licenses/fdl.txt>GNU FDL v1.3 or later</a>
+            © 2011–2018 openSUSE contributors. License: <a href=https://www.gnu.org/licenses/fdl.txt>GNU FDL v1.3 or later</a>. Visit <a href=https://github.com/kubic-project/kubic-o-o> to provide enhancements or open issues about this page.
```

Comment:
Why did you change indentation? Two spaces are enough.

Reply:
True, changed to 2 spaces only. The indendation does not seem to be consistent, other sections in the same file used 4 spaces